### PR TITLE
Add out-of-bounds read advisory for rocksdb

### PR DESCRIPTION
**Pull request by niklasf**
Draft. Will complete when the fix has landed.

### CODE REVIEW

**Review by Shnatsel — APPROVED** [review]
Thanks! Looks good to me, we'll merge once the fix is merged.

**Comment by pinkforest** [comment]
@niklasf looks like your fix was merged last month - 

However looks like it hasn't ended up into release yet as crates.io says last version six months ago

Probably waiting for a release or smth ?

**Comment by niklasf** [comment]
v0.19.0 with the fix has been released, now.

File: crates/rocksdb/RUSTSEC-0000-0000.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+date = "2022-05-11"
+url = "https://github.com/rust-rocksdb/rust-rocksdb/pull/616"
+categories = ["memory-corruption"]
+keywords = ["out-of-bounds read"]
+
+[versions]
+patched = [">= 0.19.0"]
+
+[affected]
+functions = { "rocksdb::DBWithThreadMode::open_cf_descriptors_with_ttl" = ["*"] }
```
Comment (amousset) [inline crates/rocksdb/RUSTSEC-0000-0000.md:14]:
```suggestion
functions = { "rocksdb::DBWithThreadMode::open_cf_descriptors_with_ttl" = ["< 0.19.0"] }
```

**Pull request merged**